Contributions-4-1-Ind.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Open Source Contributions
+nav_order: 2
+has_children: true
+permalink: /Contributions-4-1-Ind
+---
```

Comment:
Looks great with a straight forward and concise title: Open Source Contributions.